### ISSUE TITLE
Invisibility bug

### ISSUE THREAD

**Issue by yitzy299** [NucleoidMC/bed-wars#32]
Expected behaviour:
![image](https://user-images.githubusercontent.com/66269211/103461822-106c8280-4d19-11eb-80b9-3bcb144e1812.png)

Actual behaviour:
![image](https://user-images.githubusercontent.com/66269211/103461814-0480c080-4d19-11eb-8828-80059dca24d6.png)

The invisiblity should make armour disappear as well. I am aware this isn't vanilla behaviour however this negates the point of an invisibility potion if the player can easily be seen

**Comment by Gegy** [comment]
Yep- invisibility potions aren't added to the shop because of this. But it's not really a bug, just a feature not yet implemented.

**Comment by yitzy299** [comment]
> Yep- invisibility potions aren't added to the shop because of this. But it's not really a bug, just a feature not yet implemented.

Whoops, they were added in my PR! Shall I close for now or leave it?

**Comment by Gegy** [comment]
This is fine to keep open as a tracking issue for adding invisibility potions :)
